.github/dependabot.yml

```
@@ -37,3 +37,16 @@ updates:
     cooldown:
       default-days: 7
     open-pull-requests-limit: 10
+
+  - package-ecosystem: terraform
+    directory: ".review_apps/"
+    schedule:
+      interval: weekly
```

Can we get it to happen on Tuesdays like the other dependencies?
```suggestion
      interval: weekly
      day: tuesday
```